openmc/cmfd.py

```diff
@@ -12,12 +12,65 @@
 
 from collections.abc import Iterable
 from numbers import Real, Integral
-from xml.etree import ElementTree as ET
 import sys
-
-from openmc._xml import clean_indentation
+import numpy as np
+# Line below is added to suppress warnings when using numpy.divide to
+# divide by numpy arrays that contain entries with zero
+np.seterr(divide='ignore', invalid='ignore')
+import numpy.ctypeslib as npct
+from scipy import sparse
+import time
+from ctypes import c_int, c_double
+# See if mpi4py module can be imported, define have_mpi global variable
+try:
+    from mpi4py import MPI
+    have_mpi = True
+except ImportError:
+    have_mpi = False
+
+import openmc.capi
```

Comment:
We've been trying to avoid having a top-level `import openmc.capi` because it requires anyone who just wants to `import openmc` to have libopenmc set up in the right place. I would suggest moving the declarations of the functions from the C API (restype, argtypes, etc.) to `openmc/capi/core.py` and then have `from openmc.capi import _dll` in any function here where the C API is used.